molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/VcfValidator.java

```diff
@@ -60,7 +61,7 @@ public String validateVCF(File vcfFile)
 				logFile.createNewFile();
 			}
 
-			FileWriter fileWriter = new FileWriter(logFile.getAbsoluteFile(), true);
+			FileWriterWithEncoding fileWriter = new FileWriterWithEncoding(logFile.getAbsoluteFile(), "UTF-8", true);
```

Comment:
Why not use standard Java IO classes?
BufferedWriter out = new BufferedWriter(new OutputStreamWriter(new FileOutputStream("myfile.txt"), "UTF-8"));